api/parse.py

```diff
@@ -101,8 +31,85 @@ def FoodDatabase():
     grain = df["grain"]
     meat = df["meat"]
     fruit_veg = df["fruit_veg"]
-    print(df)
-    return dairy, grain, meat, fruit_veg
+    food_groups = np.concatenate((dairy, meat, grain, fruit_veg))
+    # print(type(food_groups))
+    #return dairy, grain, meat, fruit_veg
+    return food_groups
+
+def nlp():
+    nlp = spacy.load('en_core_web_sm')
+    matcher = PhraseMatcher(nlp.vocab, attr = 'LOWER')
+    parsed_words = parse()
+    words_string = ' '.join(parsed_words)
+    food_groups = food_database()
+    fruit_veg_string = food_groups[3]
+    #fruit_veg_string = np.array2string(food_groups[3])
+    #patterns = [nlp(text) for text in fruit_veg_string]
+    patterns = [nlp(text) for text in food_groups]
+    matcher.add("TerminologyList", None, *patterns)
```

Comment:
Some comments to explain what's going on here would be helpful (not specifically this line, just the various steps in this function)